index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Skeleton Portfolio</title>
+    <link rel="stylesheet" href="CSS/main.css">
+  </head>
+  <body>
+    <!-- header section! -->
+    <header>
+      <nav>
+        <div class="logo">
+          <a href="#"><h2>My logo</h2></a>
+        </div>
+        <div class="menu"><img src="images/Enabled.png" alt="" /></div>
+      </nav>
+    </header>
+
+    <!-- main section! -->
+    <main>
+      <!-- hello section! -->
```

Comment:
- [ ] Please, consider removing the comments from your code to make it clear - your code is talking about itself 👍

Reply:
Comments removed

Reply:
- [ ] Please remove comments as the last reviewer required.